test/e2e/broker_with_many_triggers_test.go

```diff
@@ -36,6 +36,7 @@ func DefaultBrokerCreator(_ *testlib.Client, _ string) string {
 	return sugarresources.DefaultBrokerName
 }
 
+// Deprecated, use reconciler-test based tests.
```

Comment:
Does it make sense to deprecate a test? Personally, I'd just delete it. The rekt test suite is already big enough so people have to run it to get some confidence in feature stability.

Reply:
This test is old e2e test, and what I'm doing to convert these old e2e test to rekt test  there https://github.com/knative/eventing/tree/main/test/rekt, these old e2e test we will not use them again, so once the corresponding rekt test has finished,  we will deleted them

Reply:
If the test was completely rewritten using REKT framework then it makes sense to delete the original test. Ideally, in the same PR.

Reply:
Ok I'll delete it in his PR